tpl/tplimpl/embedded/templates/shortcodes/__h_simple_assets.html

```diff
@@ -0,0 +1,35 @@
+{{ define "__h_simple_css" }}{{/* These template definitions are global. */}}
+{{/* (onedrawingperday) renamed the CSS classes to video something since everything here will be needed for both Vimeo and (hopefully) YouTube simple variants. */}}
```

Comment:
This comment will lose its value and only be confusing once this PR is merged.

Reply:
Ok. I will remove it.